Sources/TuistGraph/Models/Settings.swift

```diff
@@ -95,6 +95,8 @@ extension DefaultSettings {
     }
 }
 
+public typealias BuildConfigurationDictionary = [BuildConfiguration: Configuration?]
```

Comment:
Declaring a type is not long lengthy enough to justify the existence of another type. I'd remove it.

Reply:
as explained today in the call easier to search and find

Reply:
Plus one, and I also think it's out of scope of this PR anyway

Reply:
`extension BuildConfigurationDictionary { ... } `
is also much better to understand and read than 
`extension [BuildConfiguration: Configuration?] { ... }`
isn't it?

Reply:
removed typealias BuildConfigurationDictionary